code/modules/spacepods/construction.dm

```diff
@@ -44,7 +45,9 @@
 		var/obj/spacepod/A = new result(get_turf(holder), pod_armor)
 		A.bound_width = 64
 		A.bound_height = 64
-
+		A.armor_multiplier_applied = TRUE
+		A.max_integrity *= pod_armor.armor_multiplier
+		A.obj_integrity *= pod_armor.armor_multiplier
```

Comment:
Do not forget to copy the description (examine text) and prefix ("syndicate space pod", etc) as well!